src/ConsoleArgument.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console;
+
+use Attribute;
+use Tempest\Support\ArrayHelper;
+
+#[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_PROPERTY | Attribute::TARGET_PARAMETER)]
+final class ConsoleArgument
+{
+    /** @var string[] */
+    private array $helpMessages;
+
+    /**
+     * @param string|string[] $help
+     * @param string[] $aliases
+     */
+    public function __construct(
+        array|string $help = [],
+        public array $aliases = [],
+    ) {
+        $this->helpMessages = ArrayHelper::wrap($help);
+    }
+
+    /** @return string[] */
+    public function getHelpLines(): array
+    {
+        return $this->helpMessages;
```

Comment:
Naming is a bit confusing. We have `help` in the constructor`, `helpMessages` as the property, and then `helpLines` as the method name. I think these should all be the same.